war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -304,15 +304,17 @@ function findAncestorClass(e, cssClass) {
 function findFollowingTR(input, className) {
     // identify the parent TR
     var tr = input;
+    if(tr.tagName!=undefined && tr.parentNode!=undefined){
```

Comment:
Lack of spaces around parentheses and operators is inconsistent with the rest of the file (such as the one immediately below).

Reply:
This line of code has been removed. Since the exception handling is being done by the caller itself.